### PR TITLE
Copy Improvement Suggestions

### PR DESCRIPTION
**Pull request by x656**
Adding a link to telegram at bottom.
Adding BTC, ETH, SP500 to asset list.
Adding gov link to header.

PS: are you using yarn or npm locally, having trouble installing the deps

### CODE REVIEW

File: src/components/Footer.tsx [hunk 1]
```diff
@@ -4,6 +4,13 @@ const Footer = () => (
   <div className="flex flex-row justify-between text-xs font-monospace opacity-75">
     <span>© Cap Finance</span>
     <div className="flex flex-row space-x-12">
+      <a
```
Comment (Ulydev) [inline src/components/Footer.tsx]: I think we can only have up to three links in the footer, after that it overflows the page on mobile.

File: src/components/Header.tsx [hunk 2]
```diff
@@ -11,6 +11,13 @@ const Header = () => (
     </div>
     <div className="flex flex-row items-center space-x-8 text-xs tracking-widest">
       <a className="opacity-50">DOCS</a>
+      <a
```
Comment (Ulydev) [inline src/components/Header.tsx]: Same here, actually maybe reducing the space between links on mobile would help (space-x-4 md:space-x-8) but need to make sure it doesn't look too compact. Worst case, the cap.uly.dev client redirects to the governance so I don't think it's a big issue

File: src/components/Hero.tsx [hunk 3]
```diff
@@ -5,7 +5,7 @@ import { FaTelegramPlane } from "react-icons/fa"
 import { CSSTransition, SwitchTransition } from "react-transition-group"
 
 const AssetShowcase = () => {
-  const assets = ["AAPL", "NFLX", "GOOG"]
+  const assets = ["AAPL", "BTC", "NFLX", "GOOG", "ETH", "SP500"]
```
Comment (Ulydev) [inline src/components/Hero.tsx:8]: The underline below the assets needs to adapt to the current text, maybe we could implement that first 
Screen Shot 2021-01-24 at 21 33 10

**Review by Ulydev — CHANGES_REQUESTED** [review]
Thanks for the PR! I left some comments, might be worth discussing.

I'm using yarn, will try with npm when I get some time. Thanks again :-)

File: src/components/Footer.tsx (hunk 1 above)
Comment (x656) [reply src/components/Footer.tsx]: fair enough, removed

File: src/components/Header.tsx (hunk 2 above)
Comment (x656) [reply src/components/Header.tsx]: ok, reversed

File: src/components/Hero.tsx (hunk 3 above)
Comment (x656) [reply src/components/Hero.tsx:8]: implemented that, should be good to go

**Comment by Ulydev** [comment]
Thanks 👍

**Pull request merged**